general/1_terminal/Dima_Mayakov/Solution_task_4.txt

```diff
@@ -0,0 +1,3 @@
+To output only the texts of the message using jq
+ we need the following command: 
+$ jq '.messages[].text' result.json
```

Comment:
I'll accept this answer.

If you'd like an additional assignment, make it so the messages with links and mentions look the same as the messages without them. Currently they are rendered as arrays.